phac-nml/irida-next
# Group: Details Page Subgroups and Projects Tab

## What does this PR do and why?
_Describe in detail what your merge request does and why._

I updated the group details page to list all subgroups and projects associated with the group.

- Add icons to differentiate subgroups and projects

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

![image](https://github.com/phac-nml/irida-next/assets/11295750/a39a9f61-2a25-442a-8a88-6ad619176549)

![image](https://github.com/phac-nml/irida-next/assets/11295750/875b433f-48d1-403c-9e43-b4d10b1d1a10)

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

Starting with a default database seed:

1. Go to the[ staphylococcus group](http://localhost:3000/staphylococcus)
2. In the main area there is the new "Subgroups and projects" tab.
3. Expand and collapse different subgroups to see the project and subgroups


## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
